web-server/src/components/Settings/SyncDaysSetting.tsx

```diff
@@ -0,0 +1,137 @@
+import { FlexBox } from '../FlexBox';
+import { Box, Divider, TextField } from '@mui/material';
+import { LoadingButton } from '@mui/lab';
+import { Line } from '../Text';
+import DoneIcon from '@mui/icons-material/Done';
+import { FC, useEffect, useState } from 'react';
+import { handleApi } from '@/api-helpers/axios-api-instance';
+import { useAuth } from '@/hooks/useAuth';
+import { OrgResetBookmarkApiResponse } from '@/types/resources';
+import { useBoolState } from '@/hooks/useEasyState';
+import { useSnackbar } from 'notistack';
+import { useDispatch, useSelector } from '@/store';
+import {
+  getDefaultSyncDaysSettings,
+  updateDefaultSyncDaysSettings
+} from '@/slices/org';
+
+export const SyncDaysSetting: FC = () => {
+  const dispatch = useDispatch();
+  const inputFocus = useBoolState(false);
+  const isSaving = useBoolState(false);
+  const [syncDays, setSyncDays] = useState<number>(31);
```

Comment:
[not a required change]

```suggestion
  const syncDays = useEasyState(31);
```